signing/tests/test_secp256k1_signer.py

```diff
@@ -71,6 +77,14 @@ def test_priv_to_public_key(self):
         public_key2 = context.get_public_key(priv_key2)
         self.assertEqual(public_key2.as_hex(), KEY2_PUB_HEX)
 
+    def test_creating_private_keys_from_hex_and_bytes_to_contexts(self):
+        context = create_context("secp256k1")
+        priv_key = context.private_key_from_hex(KEY1_PRIV_HEX)
+        self.assertEqual(priv_key.as_hex(), KEY1_PUB_HEX)
```

Comment:
Copypasta: You want to assert that the private key equals `KEY1_PRIV_HEX`, not PUB